ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml

```diff
@@ -26,7 +26,7 @@ ocp4_workload_gitea_operator_starting_csv: ""
 ocp4_workload_gitea_operator_catalog_image: quay.io/gpte-devops-automation/gitea-catalog
 
 # Catalog image tag
-ocp4_workload_gitea_operator_catalog_image_tag: v1.0.3
+ocp4_workload_gitea_operator_catalog_image_tag: latest
```

Comment:
I think we always want to pin-point version, it's part of our contribution guidlines?

Reply:
Good point.